backend/Techfesia2019/Techfesia2019/settings.py

```diff
@@ -225,6 +225,16 @@
 EMAIL_HOST_PASSWORD = SENDGRID_API_KEY
 PUBLIC_ID_LENGTH = external_settings.PUBLIC_ID_LENGTH
 
+
+
+#testing. This is the settings to be used for GMAIL.
+# EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+# EMAIL_HOST='smtp.gmail.com'
+# EMAIL_PORT=587
+# EMAIL_HOST_USER='shubham.b18@iiits.in'
+# EMAIL_HOST_PASSWORD=''
```

Comment:
Please don't commit these things here. Keep them in local_settings and just put a reference to them.
If password is committed by mistake in future, it would cause problems